BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -504,3 +524,17 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
         self.fetchGoals()
     }
 }
+
+extension GalleryViewController: NSFetchedResultsControllerDelegate {
```

Comment:
I would be open to the idea of having all the various delegate interfaces implemented by `GalleryViewController` added as part of extensions, but right now _every other interface_ is included in the initial class definition, and I was confused why `NSFetchedResultsControllerDelegate` wasn't.

[Do] Let's be consistent for now and include this interface in the main class definition. We can split it out in a later PR with all the others.

```swift
class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayout, UICollectionViewDelegate, UISearchBarDelegate, SFSafariViewControllerDelegate, NSFetchedResultsControllerDelegate
```

Reply:
Made a MR for these: #615